.babelrc

```diff
@@ -1,5 +1,28 @@
 {
-  "presets": [
-    "react-native"
+  "plugins": [
+    "syntax-async-functions",
+    "syntax-class-properties",
+    "syntax-trailing-function-commas",
+    "transform-class-properties",
+    "transform-es2015-function-name",
+    "transform-es2015-arrow-functions",
+    "transform-es2015-block-scoping",
+    ["transform-es2015-classes", { "loose": true }],
+    "transform-es2015-computed-properties",
+    "check-es2015-constants",
+    "transform-es2015-destructuring",
+    "transform-es2015-parameters",
+    "transform-es2015-shorthand-properties",
+    "transform-es2015-spread",
+    "transform-es2015-template-literals",
+    "transform-es2015-literals",
+    "transform-flow-strip-types",
+    "transform-object-assign",
+    "transform-object-rest-spread",
+    "transform-react-display-name",
+    "transform-react-jsx",
+    "transform-regenerator",
+    ["transform-es2015-for-of", { "loose": true }],
+    "external-helpers"
```

Comment:
It looks like you are bringing in external helpers now, which means that folks will need to import the external helpers file in order for this to continue working for them.  This is almost certainly a breaking change, and I'm hesitant to recommend this approach for a library at this time.

Some people are discussing a modular approach to this, so that might become a good option in the future. More info: https://github.com/babel/babel/issues/5699